chef/cookbooks/ceph/recipes/osd.rb

```diff
@@ -127,7 +137,7 @@
         end
         device["device"] = d.name
         node.set["ceph"]["osd_devices"].push(device)
-        node.save
+        dirty = true
```

Comment:
We were saving here explicitly for a reason, probably becasue we are using this values `["ceph"]["osd_devices"]` immediately below.

Are we sure this is still working as before? I would say yes, as its only used in this recipe, but would like confirmation that this has been tested properly and the devices are being picked up in the recipe afterwards without saving the node.